next.config.js

```diff
@@ -24,6 +27,12 @@ const nextConfig = {
       },
     ];
   },
-};
+
+  // pages with e.g. .dev.tsx extension are only included when running locally
+  pageExtensions: ['ts', 'tsx', 'js', 'jsx'].flatMap((extension) => {
```

Comment:
this does not work because .dev.tsx extensions are still .tsx extensions, so the dev pages will still get deployed everywhere

Reply:
It works for me. I deleted `frontend/.next` directory, ran `npm run build` to create a production build and then `npm start` to run the production build. The login and logout pages weren't available and I got a 404 instead.

Reply:
wow. my mistake! i feel like i tried to do this before and ended up having to rename all the other pages but if this works fine